be/test/util/path_util_test.cpp

```diff
@@ -0,0 +1,85 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "util/path_util.h"
+
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "common/config.h"
+#include "util/logging.h"
+
+
+using std::string;
+using std::vector;
+
+namespace doris {
+
+TEST(TestPathUtil, BaseNameTest) {
+    ASSERT_EQ(".", path_util::base_name(""));
```

Comment:
should be STREQ?

Reply:
`ASSERT_EQ()` use the `==` operator to compare, so if it is wrong to use `ASSERT_EQ` on **two C-strings** because it can only test if they are in the same memory location, not if they have the same value.  i.e., only if the two string param both are C-string, `ASSERT_STREQ` should be used. 

on the other hand, If one of the two string param in `ASSERT()` is not C-string, we should use `ASSERT_EQ`, and here, the second param is an `std::string` type, so use `ASSET_EQ` is right.